crates/dock/tests/mutability_check/reassign_immutable/main.nail

```diff
@@ -0,0 +1,15 @@
+fn main() {
+    let a = 10;
+    a = 20;
```

Comment:
**Issue**: Attempt to reassign to an immutable variable 'a'. This should be prevented at compile time, but the test setup implies this is expected to compile and run, which contradicts typical Rust semantics. <br> **Fix**: If the intention is to allow reassignment, declare 'a' as mutable using 'let mut a = 10;'. Otherwise, ensure the test expects a compile-time error. <br> **Code Suggestion**: 
 ```
 +    let mut a = 10;
 +    a = 20;
 ```